packages/tools/devtools/devtools-core/src/CommonInterfaces.ts

```diff
@@ -37,12 +37,15 @@ export type FluidObjectId = string;
 
 /**
  * Base interface used in message data for events targeting a particular Fluid object (DDS) via
- * a unique ID.
+ * its {@link HasFluidObjectId.fluidObjectId | fluidObjectId}.
  *
  * @internal
  */
 export interface HasFluidObjectId {
 	/**
+	 * TODO: This should be renamed to something like 'fluidObjectKey' to better reflect its purpose as a key rather than an ID.
+	 * Cannot be changed now due to backward compatibility requirements.
+	 *
 	 * The ID of the Fluid object (DDS) associated with data or a request.
```

Comment:
```suggestion
	 * The ID of the Fluid object (DDS) associated with data or a request.
	 * @privateRemarks
	 * TODO: This should be renamed to something like 'fluidObjectKey' to better reflect its purpose as a key rather than an ID.
	 * Cannot be changed now due to backward compatibility requirements.
```